cctools/ar/archive.c

```diff
@@ -343,7 +346,8 @@ put_arobj(cfp, sb)
 			    sb->st_mode, sb->st_size, ARFMAG);
 			lname = 0;
 		} else if (lname > sizeof(hdr->ar_name) || strchr(name, ' '))
-			(void)sprintf(hb, HDR1, AR_EFMT1, (lname + 3) & ~3,
+			(void)sprintf(hb, HDR1, AR_EFMT1,
```

Comment:
I wonder if it'd be worthwhile to change `sprintf()` to `snprintf()` while you're here?